### DISCUSSION TITLE
Bottom late renders wrong, can't seem to use cadquery engine

### DISCUSSION THREAD

**Discussion by Llamalland** [joshreve/dactyl-keyboard#108] [General]
![image](https://user-images.githubusercontent.com/38714700/211180159-2b5d68f2-9463-4c6a-b55e-61af50645b82.png)

This is what I get out of openSCAD. Takes a really long time to render too for a seemingly simple object, but it's obviously not what was intended.

Also I'm not really sure how to get cadquery engine outputting -I'm on windows with the docker environment, and when I run DM-config it generates run_config.json as you'd expect, but I can't seem to get step files out of that. The documentation seems to imply that running DM-run should run dactyl_manuform.py (which is fine when I use solidpython as the engine) and that running that should also run run_config.json, but I'm not getting any .step files.

My prototype prints seem to be good enough from .scad renders from what I can tell, but I figured this might just luckily fix my first problem since there are a few settings which say they only work on cadquery.

**Comment by Michal-Szczepaniak** [comment]
i get the same, just the rim and no bottom plate :/ and when rendered the height seems to be 0?

Correction they're 0.01 high

**Comment by rek** [comment]
There was a bug introduced [here](https://github.com/joshreve/dactyl-keyboard/commit/642780436b2788e673bb40c149ee61f7f6654ce7). If you don't need those features, you can go back to this commit, which generates the base plate properly: https://github.com/joshreve/dactyl-keyboard/tree/75da53f96e389d837055921bd2eb61b58b7787e2
